.github/workflows/changelog-v3.yml

```
@@ -1,25 +1,191 @@
+        
+    - name: Get REAL validation script from v3-alpha
+      run: |
+        echo "Fetching the REAL validation script from v3-alpha branch..."
+        git fetch origin v3-alpha
+        git checkout origin/v3-alpha -- v3/scripts/validate-changelog.go
+        
+        echo "Validation script fetched successfully:"
+        ls -la v3/scripts/
+        
+    - name: Setup Go
+      uses: actions/setup-go@v4
```

actionlint (1.7.7)

the runner of "actions/setup-go@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)